### ISSUE TITLE
Validation accepts invalid SPDX YAML files

### ISSUE THREAD

**Issue by sschuberth** [spdx/spdx-online-tools#329]
For testing, I did download https://github.com/spdx/spdx-spec/blob/3d156919a06c07781bd47dc26c75efcfc9df21cd/examples/SPDXYAMLExample-2.2.spdx.yaml and changed `relationships` (plural) to `relationship` (singular) in line 636:

https://github.com/spdx/spdx-spec/blob/3d156919a06c07781bd47dc26c75efcfc9df21cd/examples/SPDXYAMLExample-2.2.spdx.yaml#L363

This deliberately renders the syntax invalid. However, uploading the broken file for validation at https://tools.spdx.org/app/validate/ shows that the SPDX document is valid:

![image](https://user-images.githubusercontent.com/349154/133752662-ea4731bc-e0d8-413d-8bf7-bfec4f38a2a9.png)

**Comment by goneall** [comment]
Good catch @sschuberth - I'll transfer this issue to the https://github.com/spdx/spdx-java-jackson-store repository which implements the JSON validation.
